pkg/testcase/cluster.go

```diff
@@ -25,16 +25,17 @@ func TestBuildCluster(g GinkgoTInterface) {
 	}
 
 	if cluster.Config.ExternalDb != "" && cluster.Config.DataStore == "external" {
-		cmd := "grep \"datastore-endpoint\" /etc/systemd/system/k3s.service"
+		cmd := fmt.Sprintf("grep \"datastore-endpoint\" /etc/rancher/%s/config.yaml", cluster.Config.Product)
 		res, err := shared.RunCommandOnNode(cmd, cluster.ServerIPs[0])
 		Expect(err).NotTo(HaveOccurred())
 		Expect(res).Should(ContainSubstring(cluster.Config.RenderedTemplate))
 
-		etcd, err := shared.RunCommandHost("cat /var/lib/rancher/k3s/server/db/etcd/config",
-			cluster.ServerIPs[0])
-		// TODO: validate also after fix https://github.com/k3s-io/k3s/issues/8744
-		Expect(etcd).Should(ContainSubstring(" No such file or directory"))
-		Expect(err).To(HaveOccurred())
+		// TODO: This fails for rke2
```

Comment:
have u forgot this ?

Reply:
Nope it actually fails for rke2 as the etcd folder is created in that path regardless, should raise a ticket and add it here

Reply:
and this 
```
// // TODO: validate also after fix https://github.com/k3s-io/k3s/issues/8744
```

you probably can remove as well